SportsHub/SportsHub.DAL/Data/Configurations/UserEntityTypeConfiguration.cs

```diff
@@ -38,7 +38,7 @@ public void Configure(EntityTypeBuilder<User> user)
 
             user.Property(x => x.Password)
                 .IsRequired(true)
-                .HasMaxLength(60)
+                .HasMaxLength(75)
```

Comment:
Try not to use magic numbers. It is better to extract them into class with constants.

Reply:
Moved to constant files inside the Domain Layer